### PR TITLE
Fix script

### PR DESCRIPTION
**Pull request by benmcmahon100**

### CODE REVIEW

**Review by VoyTechnology — APPROVED** [review]

File: importer.js [hunk 1]
```diff
@@ -20,18 +21,22 @@ fs.readFile('./users.csv', 'utf-8').then(async data => {
...
-        await page.goto('https://google.com', {waitUntil: 'networkidle'});
-        await page.goto('https://websvc.dcu.ie/clubs/socs/register', {waitUntil: 'networkidle'});
-        await page.waitForSelector('#form_id');
-        await page.click('#form_id');
-        await page.type(studentID);
-        await page.click('button[type="submit"]');
-        await page.waitForSelector('input[type="checkbox"]');
-        await page.click('input[type="checkbox"]');
-        await page.click('button[type="submit"]');
+    setTimeout(() => {
```
Comment (butlerx) [inline importer.js:24]: why do you use timeout?

File: importer.js [hunk 2]
```diff
@@ -20,18 +21,22 @@ fs.readFile('./users.csv', 'utf-8').then(async data => {
...
+      page.goto('https://google.com', {waitUntil: 'networkidle'}).then(async nothing => {
```
Comment (butlerx) [inline importer.js:25]: 1. why go back to google
1. why mix async and then you could just run `await`

File: importer.js (hunk 1 above)
Comment (benmcmahon100) [reply importer.js:24]: To make sure the auth chain finishes

Comment (benmcmahon100) [reply importer.js:24]: To make sure the auth chain finishes

File: importer.js (hunk 2 above)
Comment (benmcmahon100) [reply importer.js:25]: It wasn't happy navigating straight to the page so google was just a step that worked

File: importer.js (hunk 1 above)
Comment (butlerx) [reply importer.js:24]: why not use the wait [function](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagewaitforselectorselector-options)

**Comment by benmcmahon100** [comment]
@butlerx I intend on doing a re-write today with all that stuff. I didn't want to change the bad hacky one until we had finished doing our signups

**Comment by butlerx** [comment]
you should look #2 first as it does most of it

**Pull request merged**